subsys/net/lib/sockets/sockets_tls.c

```diff
@@ -1797,7 +1797,7 @@ static int ztls_poll_update_ctx(struct net_context *ctx,
 			/* EAGAIN might happen during or just after
 			 * DTLS handshake.
 			 */
-			if (recv(pfd->fd, NULL, 0, ZSOCK_MSG_DONTWAIT) < 0 &&
+			if (zsock_recv(pfd->fd, NULL, 0, ZSOCK_MSG_DONTWAIT) < 0 &&
```

Comment:
This commit should not be part of the PR titled "drivers/pinmux: stm32: Add i2c4 pinmux to stm32f7.h". If it's really needed, please submit it as a separate PR.